cmd/derper/derper.go

```diff
@@ -293,6 +297,7 @@ func main() {
 		})
 		if *httpPort > -1 {
 			go func() {
+
 				port80srv := &http.Server{
 					Addr:        net.JoinHostPort(listenHost, fmt.Sprintf("%d", *httpPort)),
 					Handler:     certManager.HTTPHandler(tsweb.Port80Handler{Main: mux}),
```

Comment:
won't this still redirect /generate_204 to https?

Reply:
Whoops, forgot to add/push the rest of the commit 🤦